Configuration/Settings.yaml

```diff
@@ -1,5 +1,12 @@
 Neos:
   Form:
+    FusionRenderer:
+      fusionPathPatterns:
+        Neos.Fusion:Private/Fusion: true
+        Neos.Form.FusionRenderer:Private/Fusion/Core: true
+        Neos.Form.FusionRenderer:Private/Fusion/ContainerElements: true
+        Neos.Form.FusionRenderer:Private/Fusion/Elements: true
```

Comment:
I'm just wondering: Do we really need such a fine granular configuration?
If we used a setting similar to `Neos.Neos.fusion.autoInclude` (= package based) this would get quite a bit simpler. What do you think?

Reply:
Most form are uncached, so I think it's nice is we can load just the required fusion. For two recent projet, I just need one or two new fusion object, but I load the full package, I load tons of fusion prototype that are not need for form rendering. Without a granular configuration I need to create a separate package ... that can work too. 

> get quite a bit simpler

Neos:
  Form:
    FusionRenderer:
      fusionPathPatterns:
        My.Package:Private/Form: true

It's simple ;)

@bwaidelich  So the question is more did we want a consistent configuration ...

Reply:
true. I'm fine with both

Reply:
Thinking about this one again.. Maybe we could use the glob syntax that we use for includes from Fusion, too!?